R/organization.R

```diff
@@ -59,16 +59,19 @@ download_organization_repos <- function(organizations = NULL,
 
   message(sprintf("%s archives to be saved.", length(repos)))
 
-  while (length(repos) > 0 && time_spent(start_time = start_time) < 120) {
+  while (length(repos) > 0 && time_spent(start_time = start_time) < 90) {
+    Sys.sleep(600)
     status <- purrr::map_chr(repos, function(repo) repo$migration_state)
     ready_repos <- repos[status == "exported"]
     purrr::walk(ready_repos, function(repo) repo$launch_download())
     repos <- repos[status != "exported"]
```

Comment:
One possible minor issue I see here is if a repo status changes between L64 and L66 (is this possible?). It seems slightly more robust to check which repo has `"exported"` status once at the start of the loop and re-use this variable.

Reply:
Ah no, nevermind, you're already doing this since status is computed just once. Then compared twice but it doesn't matter.